src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.tpl.html

```diff
@@ -74,7 +74,7 @@
                 </label>
             </div>
             <div>
-                <a href="" ng-style="blade.isLinkSent && {'color':'green'} || {'color': ''}" ng-click="blade.sendLink()" va-permission="platform:security:verifyEmail">
+                <a ng-if="blade.currentEntity.email === blade.origEntity.email"  href="" ng-style="blade.isLinkSent && {'color':'green'} || {'color': ''}" ng-click="blade.sendLink()" va-permission="platform:security:verifyEmail">
```

Comment:
Now it's hidden. Should it be visible, but disabled?

Reply:
There is no disabled attribute for tag "a". Can someone describe how should be shown disabled link.  @eMazeika  @Vectorfield4

Reply:
@AliveMen  The link should be gray, visible, and disabled. Dynamically add class `disabled` to the link and prevent any actions after clicking.